test/e2e/Jenkinsfile

```diff
@@ -23,7 +23,15 @@ class HarborChartFreshInstallPipelineExecutor extends FreshInstallPipelineExecut
             script.usernamePassword(credentialsId: "79e9fd98-cdf5-4f55-81fa-ecba01365534", usernameVariable: "DOCKER_HUB_USERNAME", passwordVariable: "DOCKER_HUB_PASSWORD")]) {
             script.sh """
                 # login Docker Hub to avoid the pull limit
-                docker login -u \${DOCKER_HUB_USERNAME} -p \${DOCKER_HUB_PASSWORD}
+                # docker login -u \${DOCKER_HUB_USERNAME} -p \${DOCKER_HUB_PASSWORD}
+                # set deafult registry to dockerhub.packages.vcfd.broadcom.net
+                echo '{
+                  "registry-mirrors": ["https://dockerhub.packages.vcfd.broadcom.net"]
+                }' | sudo tee /etc/docker/daemon.json > /dev/null
+
+                # Restart Docker to apply the changes
+                sudo systemctl restart docker
```

Comment:
suggest to add  `sudo systemctl reset-failed docker.service` before `sudo systemctl restart docker`